src/util/localStorageHelper.ts

```diff
@@ -0,0 +1,19 @@
+const ORIGIN_URL_NAME = 'origin_url';
+
+const setOriginUrl = (url: string) => {
+    localStorage.setItem(ORIGIN_URL_NAME, url);
+};
+
+const clearOriginUrl = () => {
+    localStorage.removeItem(ORIGIN_URL_NAME);
+};
+
+const getOriginUrl = () => {
+    return localStorage.getItem(ORIGIN_URL_NAME);
+};
+
+export {
+    setOriginUrl,
+    getOriginUrl,
+    clearOriginUrl,
```

Comment:
Naming here is not generic. You should make generic functions to util, not something specific.

addItem, removeItem getItem?